Demos/Blazorise.Demo/Layouts/MainLayout.razor

```diff
@@ -4,106 +4,126 @@
-                            <SidebarLink Toggled="@((isOpen)=> extensionsSubItemVisible = isOpen)" Visible="@extensionsSubItemVisible">
-                                <Icon Name="IconName.PuzzlePiece" Margin="Margin.Is3.FromRight" />Extensions
-                            </SidebarLink>
-                            <SidebarSubItem Visible="@extensionsSubItemVisible">
-                                <SidebarItem>
-                                    <SidebarLink To="tests/datagrid">DataGrid</SidebarLink>
-                                </SidebarItem>
-                                <SidebarItem>
-                                    <SidebarLink To="tests/components">Components</SidebarLink>
-                                </SidebarItem>
-                                <SidebarItem>
-                                    <SidebarLink To="tests/charts">Charts</SidebarLink>
-                                </SidebarItem>
-                                <SidebarItem>
-                                    <SidebarLink To="tests/livecharts">Live Charts</SidebarLink>
-                                </SidebarItem>
-                            </SidebarSubItem>
-                        </SidebarItem>
-                        <SidebarItem>
-                            <SidebarLink Toggled="@((isOpen)=> appsSubItemVisible = isOpen)" Visible="@appsSubItemVisible">
-                                <Icon Name="IconName.Smartphone" Margin="Margin.Is3.FromRight" />Apps
-                            </SidebarLink>
-                            <SidebarSubItem Visible="@appsSubItemVisible">
-                                <SidebarItem>
-                                    <SidebarLink To="apps/todo">Todo List</SidebarLink>
-                                </SidebarItem>
-                            </SidebarSubItem>
-                        </SidebarItem>
-                    </SidebarNavigation>
-                </SidebarContent>
-            </Sidebar>
+            <Bar Breakpoint="Breakpoint.Desktop" NavigationBreakpoint="Breakpoint.Tablet" ThemeContrast="ThemeContrast.Dark" Mode="BarMode.VerticalInline" CollapseMode="BarCollapseMode.Small" @bind-Visible="siderVisible">
+                <BarBrand>
+                    <BarItem>
+                        <BarLink To="">
+                            <BarIcon IconName="customIcon" />
+                            Blazorise Demo
+                        </BarLink>
+                    </BarItem>
+                </BarBrand>
+                <BarMenu>
+                    <BarStart>
+                        <BarLabel>Main</BarLabel>
+                        <BarItem>
+                            <BarLink To="">
+                                <BarIcon IconName="IconName.Dashboard" />
+                                Dashboard
+                            </BarLink>
+                        </BarItem>
+                        <BarItem>
+                            <BarDropdown Open="true">
```

Comment:
@stsrki probably worth mentioning..

As we had previously talked about, I was having a lot of trouble getting `bind-Visible` to work well with this component (due to a combination of the `BarItem` hook method **and** the custom component overrides resetting the `bind` state **and** the internal `Toggle()` method the `BarDropdownToggle` calls).

So I just added this `Open="true"` to indicate a default state. 
Obviously, this means a functionality loss now with the developer not being able to control the `Toggle` state.. but the performance and reliability are much better.

Reply:
I might look into it as I don't like having two APIs for almost the same thing. It can lead to confusion by the users.

Reply:
@stsrki okay - that would be great!
I was thinking the same, so I disabled `Visible` from being `[Parameter]` to save confusion :)